pandapower/pd2ppc_zero.py

```diff
@@ -8,17 +8,18 @@
 from itertools import product
 
 import pandapower.auxiliary as aux
-import pandas as pd
+# import pandas as pd
```

Comment:
why is this line commented-out and not removed?

Reply:
The general approach of this PR is to only correct linting issues without removing code that might be helpful for future work on the files. This import could be deleted without a problem, but there should be another PR to clean unwanted/unneeded comments anyway.

Reply:
okay